lib/matplotlib/tests/test_axes.py

```diff
@@ -1439,9 +1439,7 @@ def bump(a):
             x = 1 / (.1 + np.random.random())
             y = 2 * np.random.random() - .5
             z = 10 / (.1 + np.random.random())
-            for i in range(m):
-                w = (i / float(m) - y) * z
-                a[i] += x * np.exp(-w * w)
+            a += x * np.exp(-((np.arange(m) / m - y) * z) ** 2)
         a = np.zeros((m, n))
         for i in range(n):
             for j in range(5):
```

Comment:
Because whoever came up with that example decided to add some random numbers five times to each column rather than only once...